tests/end2end/features/test_downloads_bdd.py

```diff
@@ -167,3 +168,34 @@ def fifo_should_be_fifo(tmpdir):
     download_dir = tmpdir / 'downloads'
     assert download_dir.exists()
     assert not os.path.isfile(str(download_dir / 'fifo'))
+
+
+# TODO how to share this with test_editor_bdd.py?
```

Comment:
Move it to `tests/end2end/features/conftest.py` instead.

Reply:
Done :+1: